ynh_install_python/ynh_install_python

```diff
@@ -0,0 +1,151 @@
+#!/bin/bash
+
+# Need also the helper https://github.com/YunoHost-Apps/Experimental_helpers/blob/master/ynh_handle_getopts_args/ynh_handle_getopts_args
+
```

Comment:
```suggestion
pyenv_version=1.2.19
pyenv-virtualenv_version=1.1.5
```
Variables at the top of the script? like in [nodejs helper](https://github.com/YunoHost/yunohost/blob/7bfe564aab2300172b6aad68656dddbdd8933a27/data/helpers.d/nodejs#L3)